ratelimit/compat.py

```diff
@@ -0,0 +1,13 @@
+try:
```

Comment:
I ignored the two offending lines individually. Couldn't find any info on how to ignore a whole file via comment in `flake8` as implied by your comment; adding a `# flake8: ignore` comment to the top of the file didn't do it. This should be fine, unless you want me to do ignore this whole file via a flake8 config file at project root instead.